docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -970,6 +971,24 @@ The values of the `typescript.options.lib` compiler option should be identical t
 *Related configuration file property*: [compilerOptions](configuration-file.md#compileroptions).
 *Related API method*: [runner.compilerOptions](testcafe-api/runner/compileroptions.md)
 
+### --cache
+
+If enabled, TestCafe proxy caches webpage dependencies (like styles and scripts) for webpages visited during tests. When TestCafe revisits the page, it pulls assets from the cache instead of requesting them from the server.
+
+```sh
+testcafe chrome my-tests --cache
+```
+
+TestCafe emulates the native caching mechanism of the browser. For example, in *Chrome* TestCafe only caches resources that *Chrome* itself would cache if run without TestCafe.
+
+TestCafe caches scripts, styles, fonts, and other web assets, except for HTML. TestCafe doesn't cache HTML because that could interfere with [user roles](../guides/advanced-guides/authentication.md#user-roles). Only assets up to **5 MB** in size are stored in cache.
+
+To decrease test run time, enable `--cache` when you test web applications that require a lot of assets.
```

Comment:
Assets != requirements
```suggestion
To decrease test run time, enable `--cache` when you test asset-rich web applications.
```

Reply:
We've [been there](https://github.com/DevExpress/testcafe/pull/5969#discussion_r578197157)

Plus, assets themselves are not a problem, requesting assets is.

Reply:
```suggestion
To decrease test run time, enable `--cache` when you test web applications that load a lot of assets.
```

Reply:
"A lot" is very subjective, and refers solely to the number of assets requested by the page. The word 'heavy', on the other hand, implies that the system is strained under an increased load. The word 'rich' implies a sort of saturation. Both are a much, much better fit for this sentence than the phrase 'apps that require a lot of assets'. 

>Plus, assets themselves are not a problem, requesting assets is.

If this is the case, you should've specified that you're talking about *remote* assets in particular. 
Inline SVG images are still assets.

```suggestion
To decrease test run time, enable `--cache` when you test web applications that rely on a large amount of remote assets.
```

Reply:
As you pointed out, the number of assets is not a problem: their weight is. 

`The word 'heavy', on the other hand, implies that the system is strained under an increased load.` - that's exactly what we're going for in my opinion.

I still think this phrasing is best:

```suggestion
To decrease test run time, enable server-side caching when you test asset-heavy web applications.
```